examples/1.0.3/docker-compose-distributed-headless.yml

```diff
@@ -0,0 +1,31 @@
+version: '3'
+services:
+  master:
+    container_name: master
+    hostname: master
+    image: "grubykarol/locust:1.0.3-python3.8-alpine3.11"
+    environment:
+      ATTACKED_HOST: http://example.com
+      LOCUST_MODE: master
+      LOCUST_MASTER_BIND_PORT: 5557
+      TASK_DELAY_FROM: 5
+      TASK_DELAY_TO: 30
+      QUIET_MODE: "${QUIET_MODE:-false}"
+      LOCUST_OPTS: "--headless --expect-workers 1 -u 1 -r 1 -t 10s"
+    volumes:
+      - ./locust-scripts:/locust
+
+  slave:
+    image: "grubykarol/locust:1.0.3-python3.8-alpine3.11"
+    environment:
+      ATTACKED_HOST: http://example.com
+      LOCUST_MODE: worker
```

Comment:
from Locust 1.0.x, `slave` is replaced with `worker`, so LOCUST_MODE had to be updated accordingly